pages/upgrade.rst

```diff
@@ -28,4 +28,6 @@ For .rpm based systems :ref:`this update guide <operating_package_upgrade_rpm-yu
 Upgrading Elasticsearch
 =======================
 
+Since Graylog 2.3 Elasticsearch 5.x is supported. This Graylog version support Elasticsearch 2.x and 5.x. It is recommend to update Elasticsearch 2.x to the latest stable 5.x version, after you have Graylog 2.3 or later running. This Elasticsearch upgrade does not need to be made during the Graylog update. 
```

Comment:
Some typos:

```
Since Graylog 2.3 Elasticsearch 5.x is supported. This Graylog version supports Elasticsearch 2.x and 5.x. It is recommended to update Elasticsearch 2.x to the latest stable 5.x version, after you have Graylog 2.3 or later running. This Elasticsearch upgrade does not need to be made during the Graylog update. 
```